app/src/main/res/values/strings.xml

```diff
@@ -67,29 +67,29 @@
     <string name="recovery_code_input_hint_11">Word 11</string>
     <string name="recovery_code_input_hint_12">Word 12</string>
     <string name="recovery_code_error_empty_word">You forgot to enter this word.</string>
-    <string name="recovery_code_error_invalid_word">Wrong word. Did you mean %1$s or %2$s?</string>
-    <string name="recovery_code_error_checksum_word">Your code is invalid. Please check all words and try again!</string>
+    <string name="recovery_code_error_invalid_word">Wrong word. Did you mean \"%1$s\" or \"%2$s\"?</string>
+    <string name="recovery_code_error_checksum_word">Wrong code. Please check all the words and try again.</string>
 
     <!-- Notification -->
-    <string name="notification_channel_title">Backup Notification</string>
+    <string name="notification_channel_title">Backup notification</string>
     <string name="notification_title">Backup running</string>
-    <string name="notification_backup_result_complete">Backup complete</string>
-    <string name="notification_backup_result_rejected">Not backed up</string>
-    <string name="notification_backup_result_error">Backup failed</string>
+    <string name="notification_backup_result_complete">Done</string>
+    <string name="notification_backup_result_rejected">Could not perform backup</string>
+    <string name="notification_backup_result_error">Try again</string>
 
-    <string name="notification_success_title">Backup finished</string>
+    <string name="notification_success_title">Done</string>
```

Comment:
Mentioning the backup makes this more clearer

Reply:
True, but is that extra info ever needed? It is slower.
My understanding is that nothing is wrong, the user knows what it attempted to do, which can now just be acknowledged.

Reply:
It doesn't hurt just having it there though IMO